docs/roman_datamodels/datamodels/using_datamodels.rst

```diff
@@ -94,3 +94,42 @@ page::
 	42
 	>>> dm2.meta.exposure_start_time_mjd
 	60000.0
+
+
+.. note::
+
+	All datamodels have built in validation against the defining schemas.
+	This means that if you try to assign a value that is not allowed according
+	to one of these schemas, you will get an error. This is a good thing!
+
+	If you are getting validation errors consult the corresponding schema in
+	``rad`` to se what is allowed. If you think the schema is wrong, or you
+	continue to have issues, please contact the Roman team for help.
+
+	As a method of last resort, if you wish to turn off validation, you can do
+	so by setting the environment variable ``ROMAN_VALIDATE`` to false. This is
+	not recommended!
```

Comment:
I would add something to the effect that in this feature will be removed in the future when data models become stable.